.dockerignore

```diff
@@ -18,4 +18,19 @@ smart-contract-extensions/build/
 smart-contracts/build/
 smart-contracts/cache
 smart-contracts/artifacts
-builds/
\ No newline at end of file
+builds/
+
+shared/node_modules/
+
+.vscode
+
+# Elastic Beanstalk Files
```

Comment:
Can't wait to get rid of all this!

Reply:
yes all these aws files in locksmith can be deleted too